google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobStatistics.java

```diff
@@ -432,6 +510,47 @@ public Boolean getCacheHit() {
       return cacheHit;
     }
 
+    /**
+     * [BETA] For DDL queries, returns the operation applied to the DDL target table.
+     */
+    public String getDdlOperationPerformed() { return ddlOperationPerformed; }
+
+    /**
+     * [BETA] For DDL queries, returns the TableID of the targeted table.
+     */
+    public TableId getDdlTargetTable() { return ddlTargetTable; }
+
+    /**
+     * The original estimate of bytes processed for the job.
+     */
+    public Long getEstimatedBytesProcessed() { return estimatedBytesProcessed; }
+
+    /**
+     * The number of rows affected by a DML statement.
+     * Present only for DML statements INSERT, UPDATE or DELETE.
+     */
+    public Long getNumDmlAffectedRows() { return numDmlAffectedRows; }
+
+    /**
+     * Referenced tables for the job.
+     * Queries that reference more than 50 tables will not have a complete list.
+     */
+    public List<TableId> getReferencedTables() { return referencedTables; }
+
+    /**
+     * [BETA] The type of query statement, if valid.
+     * Possible values include:
+     * SELECT
+     * INSERT
+     * UPDATE
+     * DELETE
+     * CREATE_TABLE
+     * CREATE_TABLE_AS_SELECT
+     * DROP_TABLE
+     * CREATE_VIEW
+     * DROP_VIEW
+     */
+    public String getStatementType() { return statementType; }
```

Comment:
@pongad Should we make an enum for statement type? I've noticed enums for other things in the Java client.

Reply:
I think enums are appropriate here.